### PR TITLE
Updated README for develop/test guide

### PR DESCRIPTION
**Pull request by brgnepal**
Updated with few missing instructions needed for getting start with development/test.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,17 +37,23 @@ The [Atomic Developer Bundle](https://github.com/projectatomic/adb-atomic-develo
...
 
-4. Develop.  You can test the command by running `bundle exec vagrant service-manager`
+3. `cd vagrant-service-manager`
 
-5. When you are ready to build the release, get a maintainer to:
+4. Run `bundle install`
+
+5. Start the box with `bundle exec vagrant up`
+
+6. Develop. Test the command by running `bundle exec vagrant service-manager`
```
Comment (navidshaikh) [inline README.md]: re-phrase it like
6. Develop and test the command by  running `bundle exec vagrant service-manager`

Comment (brgnepal) [inline README.md]: I thought that initially then taken back as it looks like `bundle exec vagrant service-manager` command is used for both develop and test.

Comment (navidshaikh) [inline README.md]: I think there is separation using "and"

Comment (brgnepal) [inline README.md]: Done.

**Comment by brgnepal** [comment]
@navidshaikh Done with your changes and suggested changes from Preeti.

**Comment by navidshaikh** [comment]
LGTM!
 @LalatenduMohanty @bexelbie ping

File: README.md (hunk 1 above)
Comment (LalatenduMohanty) [inline README.md]: @budhrg I think it should be a comma between Develop, test  and s/Test/test/

**Comment by LalatenduMohanty** [comment]
@budhrg any specific reason for having two commits in the PR, can it be one commit?

File: README.md [hunk 2]
```diff
@@ -37,17 +37,23 @@ The [Atomic Developer Bundle](https://github.com/projectatomic/adb-atomic-develo
   * IRC: #atomic and #nulecule on freenode
   * Mailing List: container-tools@redhat.com
 
-## How to Develop/Test
+## How to Develop/Test?
```
Comment (bexelbie) [inline README.md]: Looks like a ? got added by mistake

File: README.md [hunk 3]
```diff
@@ -59,19 +65,22 @@ The [Atomic Developer Bundle](https://github.com/projectatomic/adb-atomic-develo
 
   5. Create a Github release
 
-###How to build the Vagrant plugin using Bundler
+### How to build the Vagrant plugin using Bundler?
```
Comment (bexelbie) [inline README.md]: Same here, we don't seem to need the ?

File: README.md [hunk 4]
```diff
@@ -59,19 +65,22 @@ The [Atomic Developer Bundle](https://github.com/projectatomic/adb-atomic-develo
...
 
 You can also use Bundler to build the plugin and install it manually in your Vagrant environment
 
-```
-git clone this repository and run below commands inside the repository
+Run the below commands inside the repository:
```
Comment (bexelbie) [inline README.md]: Run the commands below inside of the repository:

File: README.md [hunk 5]
```diff
@@ -59,19 +65,22 @@ The [Atomic Developer Bundle](https://github.com/projectatomic/adb-atomic-develo
...
 
+```
```
Comment (bexelbie) [inline README.md:74]: Since this is now a shell example, can you make $ or # prompts and not numbers?

**Comment by bexelbie** [comment]
+1 for squashing

Add update note for changelog please

**Comment by brgnepal** [comment]
@navidshaikh, @LalatenduMohanty , @bexelbie  Done with squashing and suggested changes.

**Comment by bexelbie** [comment]
LGTM

**Comment by LalatenduMohanty** [comment]
LGTM

**Pull request merged**